File: 4.Clients/WebApi/ClientApp/src/entities/technical-stage.ts
import { StageStatusEnum } from './enums/stage-status.enum';
import { SeniorityEnum } from './enums/seniority.enum';
import { Stage } from './stage';

export class TechnicalStage extends Stage{
    seniority: SeniorityEnum;
    client: string;
}

File: 1.Domain/Domain.Model/TechnicalStage.cs
using Domain.Model.Enum;
using System;
using System.Collections.Generic;
using System.Text;

namespace Domain.Model
{
    public class TechnicalStage: Stage
    {
        public Seniority Seniority { get; set; }
        public Seniority AlternativeSeniority { get; set; }
        public string Client { get; set; }
    }
}
